bin/seal.rb

```diff
@@ -3,5 +3,7 @@
 require_relative '../lib/seal'
 require_relative '../lib/team_builder'
 
-teams = TeamBuilder.build(env: ENV, team_name: ARGV[0])
-Seal.new(teams).bark(mode: ARGV[1])
+if !Date.today.saturday? && !Date.today.sunday?
```

Comment:
I'll update the commit message but basically, by having it in SlackPoster it still makes all the API calls, etc, it just won't post to slack, so that's a lot of unnecessary work.